src/main/disemvowel.js

```diff
@@ -0,0 +1,7 @@
+const disemvowel = str => {
+  return str.replace(/[aeiou]/ig,'').replace(/\s+/ig,'')
```

Comment:
you could also place the \s inside the vowels regex statement if you want to. something like 

str.replace(/[\saeiou]/ig,'')